### DISCUSSION TITLE
Dolphin (probably) being an idiot

### DISCUSSION THREAD

**Discussion by Melechtna** [v1993/evdevhook#17] [Q&A]
I'm not sure if it's on evdev, or on dolphin, but, it works just fine, even detects motion correctly, right up until I start literally any thing. Then it just stops working all together.

**Comment by v1993** [comment]
What controller are you using? Does this happen both with GC and Wii games?

**Reply by v1993** [reply]
Also, it seems that I can reproduce this locally. I'll hopefully take a look soon. Thanks for reporting!

**Reply by Melechtna** [reply]
Given I don't see the point in testing a Gamecube game, I haven't tested, yes it does. It breaks even then. It's a PS5 with the added fix for correct gyro in the json. I also tested it with Ryujinx, and it doesn't work at all there either.

Nobara Linux (Basically Fedora) before you ask.

![Screenshot_20230105_235445](https://user-images.githubusercontent.com/81296467/210939257-30aa79f4-66fb-483d-ad79-975fa4b6f43b.png)

**Reply by v1993** [reply]
Given that my other motion server (linuxmotehook2) does not have this problem, I probably should just rewrite evdevhook to use the same underlying library (gcemuhook). Current solution is rather dirty in some regards anyways and there's essentially no reason to sick to it.
